projects/storefrontapp-e2e-cypress/cypress/integration/regression/wish-list.e2e-spec.ts

```diff
@@ -0,0 +1,50 @@
+import * as wishlist from '../../helpers/wish-list';
+
+describe('Wish list', () => {
+  before(() => {
+    cy.window().then(win => {
+      win.localStorage.clear();
+    });
+  });
+
+  describe('Anonymous', () => {
+    it('should go to login and redirect to PDP', () => {
+      wishlist.addToWishListAnonymous(wishlist.products[0]);
+    });
+  });
+
+  describe('Logged in', () => {
+    beforeEach(() => {
```

Comment:
move the saves for storage at the global level, because it fails once it moves from anonymous to log in (since the user was not logged in)

put it right after the before clause for clearing the storage